src/OrganisationRegistry.SqlServer/Organisation/OrganisationBankAccountListItemView.cs

```diff
@@ -154,6 +155,24 @@ public async Task Handle(DbConnection dbConnection, DbTransaction dbTransaction,
             }
         }
 
+        public async Task Handle(DbConnection dbConnection, DbTransaction dbTransaction, IEnvelope<OrganisationCouplingWithKboCancelled> message)
+        {
+            if (!message.Body.OrganisationBankAccountIdsToCancel.Any())
```

Comment:
Should you have a bunch of logic here with `BeforeKboCoupling` to restore bank accounts from before coupling?

Reply:
Nevermind, you are passing in explicit kbo id's 👍